Gaming The Syllabus/Assets/XML/Abilities/PlayerAbilities.xml

```diff
@@ -23,7 +23,7 @@
     <hitMessage> Used Fire Blast and dealt </hitMessage>
     <missMessage> Tried using Fire Blast but missed! </missMessage>
   </PlayerAbility>
-  <PlayerAbility name="Super Attack" type="attack" default="false">
+  <!-- <PlayerAbility name="Super Attack" type="attack" default="false">
```

Comment:
Should this be deleted since it's commented out? Or is it temporarily left out for some reason and will be added back later?